chapter_3/irb_calculator.md

```diff
@@ -0,0 +1,105 @@
+(104389658_chapter_3_master) Serg09
+---
+Violettas-MacBook-Pro:lrthw_exercises $ irb
+---
+
+    2.2.3 :001 > 25+30/6
```

Comment:
Doesn't matter as much in irb, but best practice is to have spaces around operators (`25 + 30 / 6` vs (`25+30/6`).